SingleSource/UnitTests/AArch64/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
+  if(ARCH STREQUAL "AArch64")
+    llvm_singlesource(PREFIX "aarch64-")
```

Comment:
On Linux we also need to specify ``--rtlib=compiler-rt`` to make FMV enabled in driver

Reply:
Ok. I'll take out the `#if __linux__` for now, and add an Apple check here for now.